netmiko/nokia/nokia_sros.py

```diff
@@ -286,8 +321,12 @@ def check_file_exists(self, remote_cmd=""):
                 raise ValueError("Unexpected output from check_file_exists")
         elif self.direction == "get":
             return os.path.exists(self.dest_file)
+        else:
+            raise ValueError("Unexpected value for self.direction")
 
-    def remote_file_size(self, remote_cmd=None, remote_file=None):
+    def remote_file_size(
+        self, remote_cmd: Optional[str] = None, remote_file: Optional[str] = None
```

Comment:
I would switch this to be `remote_cmd: str = ""` to make it consistent with what is in `scp_handler` for remote file size.